tests/ci_visibility/api/fake_runner_all_fail.py

```diff
@@ -4,123 +4,124 @@
 from pathlib import Path
 import sys
 
-from ddtrace.ext.ci_visibility import api
+from ddtrace.ext.test_visibility import api
+import ddtrace.ext.test_visibility.item_ids
```

Comment:
nit: Should we alias this to be shorter?

Reply:
I guess it's just a test file, so it would really just be for internal readability 😅

Reply:
Same as for the other comment. Maybe in another PR.